Source/OCcore/RawDump/rawdump.h

```diff
@@ -0,0 +1,12 @@
+//
+// Created by supragya on 13/4/18.
+//
+
+#ifndef APERTUSOPENCINE_RAWDUMP_H
```

Comment:
No need to give it that long names, in my opinion, just file name is ok

Reply:
corrected in #38, RAWDUMP_H